### DISCUSSION TITLE
Welcome to Novel Engine discussions!

### DISCUSSION THREAD

**Discussion by dotargz** [dotargz/novel-engine#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by devxan** [comment]
I'm [Xan](https://xan.lol/) and I'm going to supply a ton of Feature Requests to totally bloat this engine! (sarcasm; hopefully it will run fast.) 

I hope that it will be easily configurable so that anyone will be able to create their own interactive fiction/choose your own adventure/visual novel game for the web! (Or any other platform. I want that json to work without php, like with another language)
